tutorial_boosting.md

```diff
@@ -64,8 +68,12 @@ python search_with_boosting.py
 
 Now you can check ```results[]```, the products that correspond to the boost condition became reranked.
 
-Please note that setting boost strength to 1.0 gives the item a strong promotion. However, it does not necessarily mean that the boosted item will be the top result at all times, nor that other items will be excluded. 
-Results could still be shown even when none of them matches the condition. Also, results that are significantly more relevant to the search query can still trump your heavily favored but irrelevant items.
+## Some notes about boosting
+
+Please note that setting boost strength to 1.0 gives the item a strong promotion. However, it **does not necessarily mean that the boosted item will be the top result at all times**, nor that other items will be excluded. 
```

Comment:
Suggestion: add an article before the "boost strength".

Please note that setting a boost strength to 1.0 gives the item a strong promotion. However, it **does not necessarily mean that the boosted item will be the top result at all times**, nor that other items will be excluded.

Reply:
done